pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -199,6 +199,13 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				node.Name, task.Namespace, task.Name)
 		}
 
+		// Predicate checks whether enough resource is available for the task in that node
+		clonedNode := node.Clone()
+		if !task.Resreq.LessEqual(clonedNode.Idle.Add(node.Releasing)) {
+			return fmt.Errorf("task <%s/%s> ResourceFit failed on node <%s>",
+				node.Name, task.Namespace, task.Name)
```

Comment:
`node.Name` should be put at last variable

Reply:
done